SUPPORT.md

```diff
@@ -0,0 +1,45 @@
+# Support for GradleUp projects
+
+Any bug reports, feature requests and comments are welcome!
+The project keeps evolving,
+and any feedback from end users and developers will be appreciated.
+At the moment, there is no public Slack channel,
```

Comment:
Nit: no need to mention it if it's not there. This way this paragraph will still be true when we add the channel.

```suggestion
and any feedback from end users and developers will be appreciated.
```